config/locales/en/nsm/steps.yml

```diff
@@ -40,6 +40,10 @@ en:
         edit:
           page_title: Case details
           heading: Case details
+      case_category:
+        edit:
+          page_title: Case category
```

Comment:
Nit: Is this page title in the requirements? In general I think we want our page titles to be the same as our h1s unless specified otherwise (can't find that spelled out in GDS, but in HMRC they make that link: https://design.tax.service.gov.uk/hmrc-design-patterns/page-title/)